### ISSUE TITLE
`clean-repo-tabs`: menuItem is undefined

### ISSUE THREAD

**Issue by cheap-glitch** [refined-github/refined-github#7140]
### Description

```
onlyShowInDropdown@moz-extension://49f9749d-a211-4327-ae0d-6e28fb7a9a1c/assets/refined-github.js:12081:7
init@moz-extension://49f9749d-a211-4327-ae0d-6e28fb7a9a1c/assets/refined-github.js:12142:27
```

The feature throws an error on those lines:
https://github.com/refined-github/refined-github/blob/a0381ca315606a53def92932ed7927dca84d8963/source/features/clean-repo-tabs.tsx#L43-L44

### How to replicate the issue + URL

1. Open https://github.com/refined-github/sandbox/pull/12/checks
2. Click on the "Files" tab

### Extension version

23.12.1

### Browser(s) used

Firefox

**Comment by fregante** [comment]
If it happens only after a navigation, it's probably due to:

- https://github.com/refined-github/refined-github/issues/6437

I should really look into those AJAX issues

**Comment by fregante** [comment]
Closing in favor of https://github.com/refined-github/refined-github/issues/6437

**Comment by fregante** [comment]
I was wrong, https://github.com/refined-github/refined-github/pull/7513 does not fix it

**Comment by fregante** [comment]
Duplicate of https://github.com/refined-github/refined-github/issues/7806

**Comment by fregante** [comment]
Again unrelated 😂
